api/views/search.py

```diff
@@ -0,0 +1,30 @@
+from api import app, db
+from api.models import Story, StoryPOI, POI
+from api.utils import create_response, row_constructor
+from flask import Blueprint, request
+
+SEARCH_URL = "/search"
+
+mod = Blueprint('search', __name__)
+
+@app.route(SEARCH_URL, methods=['GET'])
+def search_pois():
+    data = request.args
+    pois = []
+    pois1 = []
+    pois2 = []
+
+    if data.get('q') is not None:
+        if data.get('name') is not None:
+            if data.get('name') == "True":
+                pois1 = POI.query.filter(POI.name.contains(data['q']))
+        if data.get('description') is not None:
+            if data.get('description') == "True":
+                pois2 = POI.query.filter(POI.description.contains(data['q']))
+
+    if pois1.count() == 0 and pois2.count() == 0:
+        return create_response(status=404, message='No POIs found')
```

Comment:
so this was an oversight from before, but we realized it's wrong to `404` when there's no pois. it's certainly valid to have no pois or to have no matches during a search, so it should send a `200` and an empty list